ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRaoModeller.java

```diff
@@ -59,14 +59,14 @@ public void updateProblem(Network network, SystematicSensitivityAnalysisResult s
         fillerList.forEach(AbstractProblemFiller::update);
     }
 
-    public RaoComputationResult solve() {
+    public RaoComputationResult solve(String resultVariantId) {
```

Comment:
Indeed it is required for now, but maybe we should discuss about that later if it's what we want or not

Reply:
Yeah for now we need it for the post processors (to update the range action result extensions). Not sure how to avoid this (unless we find a smarter way of storing the variant ids)